massa-models/src/serialization_context.rs

```diff
@@ -72,13 +72,43 @@ pub struct SerializationContext {
     pub max_bootstrap_pos_entries: u32,
     pub max_bootstrap_message_size: u32,
 }
-
+#[cfg(not(feature = "sandbox"))]
 impl Default for SerializationContext {
     fn default() -> Self {
         SerializationContext::const_default()
     }
 }
 
+#[cfg(feature = "sandbox")]
```

Comment:
Could you comment what's the diff with the default with not sandbox

Reply:
Done.